src/ensembl/src/content/app/browser/browser-bar/BrowserBar.test.tsx

```diff
@@ -56,21 +53,21 @@ describe('<BrowserBar />', () => {
       const renderedBrowserBar = mount(
         renderBrowserBar({ ensObject: createEnsObject('gene') })
       );
-      expect(renderedBrowserBar.find(GeneSummaryStrip).length).toBe(1);
+      expect(renderedBrowserBar.find(FeatureSummaryStrip).length).toBe(1);
```

Comment:
description of the test (string in the `test` function) needs to be updated accordingly